ega-charts/localega/templates/keyserver-deploy.yaml

```diff
@@ -49,9 +49,9 @@ spec:
               name: {{ template "localega.name" . }}-db-secret
               key: postgres_password
         - name: DB_URL
-          value: "jdbc:postgresql://{{ if .Values.config.postgres_host }}{{ .Values.config.postgres_host | quote }}{{ else }}{{ template "localega.fullname" . }}-db{{ end }}/{{ .Values.config.postgres_db_name }}"
+          value: "jdbc:postgresql://{{ if .Values.config.postgres_host }}{{ .Values.config.postgres_host }}{{ else }}{{ template "localega.fullname" . }}-db{{ end }}/{{ .Values.config.postgres_db_name }}"
```

Comment:
we might want to add an option to the jdbc for ssl mode similar to the Python services: https://jdbc.postgresql.org/documentation/head/connect.html

Reply:
Coming with the work to move over to ega-archive built images.